elastic/Makefile

```diff
@@ -59,7 +59,7 @@ install: venv-create
 	# install pytest for tests
 	. $(VENV_ACTIVATE_FILE); pip3 install pytest==6.2.5 pytest-benchmark==3.2.2
 	# install dependencies for tests
-	. $(VENV_ACTIVATE_FILE); pip3 install geneve==0.0.3 pytest-asyncio==0.18.1
+	. $(VENV_ACTIVATE_FILE); pip3 install geneve==0.0.3 pytest-asyncio==0.18.1 'https://github.com/elastic/package-assets/archive/main.tar.gz'
```

Comment:
What are your thoughts on using `git+https://github.com/elastic/package-assets` instead? It fetches the package from git instead of what appears to be an obscure undocumented archive? This is basically how we install Rally below.

cc @cavokz

Reply:
Can the version (branch/revision) be pinned with `git+https`?

Reply:
fixed in [6f82c67](https://github.com/elastic/rally-tracks/pull/295/commits/6f82c6781911d23286f880dfd9851e6c9e7ce2a5)